Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -254,6 +254,21 @@ - (void)authorizationController:(ASAuthorizationController *)controller didCompl
     return requests;
 }
 
+void WebAuthenticatorCoordinatorProxy::pauseConditionalAssertion()
+{
+    m_paused = true;
+    if (m_isConditionalAssertion)
+        [m_controller cancel];
+}
+
+void WebAuthenticatorCoordinatorProxy::unpauseConditionalAssertion()
+{
+    if (m_paused && m_controller && m_isConditionalAssertion)
+        [m_controller performAutoFillAssistedRequests];
+
+    m_paused = false;
```

Comment:
Should we clear `m_isConditionalAssertion` here, too?

Reply:
Not here or we couldn't pause again. I'll clear it in the non-paused case of the completion handler.